journal.py

```diff
@@ -40,10 +59,15 @@ def init_db():
     Base.metadata.create_all(bind=engine)
 
 
-@view_config(route_name='home', renderer='string')
-def home(request):
-    import pdb; pdb.set_trace()
-    return "Hello World"
+@view_config(route_name='home', renderer='templates/list.jinja2')
+def list_view(request):
+    entries = Entry.all()
+    return {'entries': entries}
+
+
+#@view_config(route_name='home', renderer='string')
+#def home(request):
+#    return "Hello World"
```

Comment:
No need for this to be here in a pull request. You can always return to it by going through your git history.